sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,328 @@
+- Added README.md for the Unified Runtime directory. [453ffdcf41dd]
+- Updated the versioning section in [`ABIPolicyGuide`](doc/developer/ABIPolicyGuide.md). [dfe6e4e542ab]
+- Added `invoke_simd` section to [`README`](sycl/doc/extensions/experimental/sycl_ext_intel_esimd/README.md) and updated [sycl_ext_intel_esimd](doc/extensions/experimental/sycl_ext_intel_esimd/sycl_ext_intel_esimd.md) [52567e8cd428]
+- Updated [`sycl_ext_oneapi_device_architecture`](doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc) and [`sycl_ext_oneapi_device_if`](doc/extensions/proposed/sycl_ext_oneapi_device_if.asciidoc) to allow captures in `if_device_has`, `if_architecture_is`. [acfc905a994a]
+- Updated [SPV_INTEL_joint_matrix](doc/design/spirv-extensions/SPV_INTEL_joint_matrix.asciidoc) extension. [adbbf0f39c25]
+
+## Bug fixes
+
+### SYCL Compiler
+- Fixed `uses_aspects` to be applied to function declarations and not only function definitions. [5d928976f9e2]
+- Fixed `ivdep` attribute in template contexts. [008e3743929e]
+- Fixed option restriction for device with Windows. [be3482ff3f0e]
+- Fixed handling unsupported attributes for ESIMD. [8990c5503d47]
+- Fixed optimization option processing for device options. [9a82d283ae2b]
+- Enabled proper behavior of optional kernel features with SYCL_EXTERNAL. [1dce70f413e6]
+- Fixed "SLM init call is supported only in kernels" message for `slm_init()` inside a ESIMD kernel when compiling with `-O0` option. [2056c669b122]
+- Fixed memory leak in `Base64::decode`. [28113ec69167]
+- Fixed crash when using libraries with `-fsycl`. [ef0d151e57a4]
+- Added predefines for `/MDd` to fix certain build issues when using reductions on Windows. [1a602d77b7b2]
+- Fixed forward declaration in integration header. [9eeb7825ea2d]
+- Fixed an issue with setting `VCSLMSize` attribute which was causing JIT compilation failure when using `slm_init()` in ESIMD kernel.  [f24a0bb343b2]
+- Fixed option for undefined symbols for hostdep link to work with gold linker. [554ed2187d32]
+- Fixed directory field of `DIFile`. [797187de2199]
+
+### SYCL Library
+- Fixed the bug in the Level zero backend where `zeModuleDestroy` is called although `ext::oneapi::level_zero::ownership::keep` is passed to `make_kernel_bundle`` leading to a double free corruption. [1344fa29e6eb]
+- Addressed specification mismatches for `multi_ptr`. [ed5941feb363]
+- Fixed corner case when using short or char with exclusive scan. [aa31c1e2deb9]
+- Updated the legacy `multi_ptr` construction from local_accessors to conform with the construction method used by other accessors. [6314af860c8c]
+- Made `get_pointer` noexcept to comply with SYCL 2020. [85d84114510c]
+- Started to return the correct `sycl::errc` in case of invalid `sycl::reqd_work_group_size`. [d16a1fd39c08]
+- Fixed `sycl::remquo` truncation error. [43fbde38e26f]
+- Fixed `std::true/false_t` to `std::true/false_type` in device aspects traits implementation. [a0d55542df0e]
```

Comment:
Not sure if we need to mention that. It was a bugfix made during `any_device_has`/`all_devices_have` implementation and there may not be a state where the issue is discoverable. Tagging @maarquitos14

Reply:
Removed.